sdk/include/reactos/layout.h

```diff
@@ -67,6 +67,16 @@ _layout_MoveGrip(LAYOUT_DATA *pData, HDWP hDwp OPTIONAL)
 static __inline void
 LayoutShowGrip(LAYOUT_DATA *pData, BOOL bShow)
 {
+    UINT uSWP_ = SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE |
```

Comment:
What's the point of the underscore after the variable name?

Reply:
OK. Underline is deleted.